README.md

```diff
@@ -28,6 +28,7 @@ Table of Contents (ToC)
 * Storage Options
   * [Local](docs/Local.md)
   * [S3](docs/S3.md)
+  * [R2](docs/S3.md#R2)
```

Comment:
I'm not sure if we need separate section for all s3-compatible services. Maybe inside `S3.md` is enough?

Leave this for maintainers @sylvestre @drahnr to decide.

Reply:
Probably not needed here, but maybe worth changing `S3` to `S3 and compatible` or something similar?

Reply:
Nice idea to me.

Reply:
sounds good

Reply:
> maybe worth changing `S3` to `S3 and compatible` or something similar?

Please do this change, others LGTM~